### DISCUSSION TITLE
Playwright Driver

### DISCUSSION THREAD

**Discussion by Nezteb** [keathley/wallaby#753] [Ideas]
It'd be cool if Wallaby supported [the Playwright testing/automation framework](https://github.com/microsoft/playwright) as an alternative to Puppeteer and Selenium. [Playwright is briefly mentioned in another GH discussion](https://github.com/elixir-wallaby/wallaby/discussions/643), but not in a driver context.

**Comment by drselump14** [comment]
would love to have this feature
